source/process/community-process.rst

```diff
@@ -0,0 +1,101 @@
+Community Monitoring Process
+============================
+
+This document outlines the process for monitoring and responding to community questions in various channels.
+
+Community Product Manager
+--------------------------
+
+Every week, we rotate which PM is responsible for making sure community questions get answered. We rotate alphabetically (for example, Eric -> Jason -> Lindsay).
+
+When responding to issues:
+
+1. Identify what type of post it is:
+
+  - Troubleshooting
+  - Feature Request
+  - Bug
+  - Developer/Product Question
+
+2. Ask questions to narrow down or understand the issue better
+
+3. Respond:
+
+  - Look for related documentation, Jira ticket, feature request, etc.
+  - If there is no related documentation:
+
+    - For reproducible bugs, create a Jira ticket
+    - For product questions or troubleshooting, update docs.mattermost.com if appropriate
+    - For feature requests, ask them to `create a feature request <https://www.mattermost.org/feature-ideas/>`_ and post the link back
+
+  - In your response, link to the documentation
```

Comment:
I think point 1 was saying find the documentation if it exists, point 2 is saying write the documentation if it doesn't exist, and point 3 is saying in your response, include the link from point 1 or 2. 

If it's confusing I can combine some points if you have suggestions, but I think it's important to explicitly state the link should be included.